src/lib/gql/zeusHasuraAdmin/reactQuery.ts

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable */
+
+import {
+  ValueTypes,
+  GraphQLTypes,
+  InputType,
+  Chain,
+  OperationOptions,
+  chainOptions,
+} from './index';
+import { useMutation, useQuery } from 'react-query';
+import type { UseMutationOptions, UseQueryOptions } from 'react-query';
+
+export function useTypedMutation<
+  O extends 'mutation_root',
+  TData extends ValueTypes[O],
+  TResult = InputType<GraphQLTypes[O], TData>
+>(
+  mutationKey: string | unknown[],
+  mutation: TData | ValueTypes[O],
+  options?: Omit<UseMutationOptions<TResult>, 'mutationKey' | 'mutationFn'>,
+  zeusOptions?: OperationOptions,
+  host = 'http://localhost:8080/v1/graphql',
```

Comment:
Can we swap these to use $HASURA_URL? (This will make the Hasura Host canonical (defined in only one place) and let me run the app on 8081 too)

Reply:
I needed to write a sed mini script to change, because this was autogenerated

Reply:
Hmm, is this going to be a problem again (hardcoded URLs) when we regenerate this with the zeus gen script?

Reply:
this isn't going to be a problem; in the app, we can use a shim function that sets the host correctly, and in the serverless functions, these hooks aren't used